### ISSUE TITLE
[A11yMAS][Accessibility]High Contrast][MAS4.3.1]Eclipse][Wipro]:Button text is not visible present in run on web app for containers in hc black mode

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1420]
**Repro Steps:**
1. Open and Start Eclipse.
2. Create a new dynamic web project.
3. Navigate to and select Project ->Azure->Publish as Azure web app for containers.
4. "Run on web app for containers" dialog , tab to "Azure Container Registry" button and observe the button text in both High Contrast modes.
Same issue is also observed for "Web App for Container" "Resource Group" and "App Service Plan" buttons

**Expected Result:**
Button text should be visible in HC Black mode

**Actual Result:**
Button text is not visible in HC Black Mode

**Platform:**
**Eclipse Version:**
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
**OSBuild Version:** Version 1709(Os Build 16299.192)

![highlighted button text is not visible in HCBlack theme in run on webapp for containers](https://user-images.githubusercontent.com/22956130/35717697-ddf239b2-0806-11e8-8d04-ea130e3f0ccc.jpg)

**Comment by Eskibear** [comment]
Determined by Eclipse IDE, cannot be fixed here.

**Comment by Eskibear** [comment]
`ExpandBar` limitation.

**Comment by Eskibear** [comment]
* **Importance of experience**: minimal. It's a legacy feature hardly ever used.
* **Quality of experience**: not impacted. All the widgets are expanded by default, users can complete the scenario without expand/collapse any of them. Users can also use the Azure portal to deploy.
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

#### Workaround
![image](https://user-images.githubusercontent.com/2351748/47148956-c0a26c00-d304-11e8-9064-bf67314627d2.png)

**Comment by umasarath52** [comment]
Discussed with Ryan and approved to close as this scenario is non blocking and low impact.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749868
